lib/awful/widget/launcher.lua

```diff
@@ -1,4 +1,12 @@
 ---------------------------------------------------------------------------
+--  A button widget which hosts a menu or starts a command
```

Comment:
Don't miss the final dot `.` on summaries. It breaks ldoc.
```suggestion
--  A button widget which hosts a menu or starts a command.
```

Reply:
Done